2019-11-12 02:14:53.174 - [Info] MITM Version: 1.26
2019-11-12 02:14:53.180 - [Info] Auto Access Enabled: true
2019-11-12 02:14:53.181 - [Debug] [Init] Auto Access Barrier: true
2019-11-12 02:14:53.206 - [Info] SSH man-in-the-middle server for 172.20.0.6 listening on 0.0.0.0:10105
2019-11-12 02:24:39.325 - [Debug] [Connection] Attacker connected: 51.91.251.177 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-12 02:24:39.898 - [Debug] [Auth] Attacker 51.91.251.177 trying to authenticate with "password"
2019-11-12 02:24:39.901 - [Debug] [Auto Access] Attacker: 51.91.251.177, Threshold: 1, Attempts: 1
2019-11-12 02:24:39.901 - [Debug] [Auto Access] Compromising the honeypot
2019-11-12 02:24:39.901 - [Debug] [Auto Access] Adding the following credentials: "mysql:rascal"
2019-11-12 02:24:40.368 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-12 02:24:40.369 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.6
2019-11-12 02:24:40.444 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-12 02:24:40.800 - [Debug] [Connection] Attacker connected: 51.91.251.177 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-12 02:24:41.364 - [Debug] [Auth] Attacker 51.91.251.177 trying to authenticate with "password"
2019-11-12 02:24:41.365 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-12 02:24:41.367 - [Debug] [Auth] Attacker: 51.91.251.177 has so far made 1 attempts. Remaining: 5 attempts
2019-11-12 02:24:41.565 - [Debug] [Connection] Attacker closed the connection
2019-11-12 02:24:41.726 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
